### PR TITLE
Feature/vault documentation

### PR DESCRIPTION
**Pull request by erzetpe**

### CODE REVIEW

**Comment by erzetpe** [comment]
This is documentation for issues: #1191, #1190.

File: docs/home/howto/SECURITY.md [hunk 1]
```diff
@@ -367,3 +367,257 @@ cp /etc/kubernetes/admin.conf $HOME/
...
+```yaml
+kind: configuration/vault
+title: Vault Config
+name: default
+specification:
+  vault_enabled: true # enable Vault install
+  vault_system_user: vault # user name under which Vault service will be running
+  vault_system_group: vault # group name under which Vault service will be running
+  enable_vault_audit_logs: false # turn on audit logs that can be found at /opt/vault/logs/vault_audit.log
+  enable_vault_ui: false # enable Vault UI, shouldn't be used at production
```
Comment (mkyc) [inline docs/home/howto/SECURITY.md:394]: why?

File: docs/home/howto/SECURITY.md [hunk 2]
```diff
@@ -367,3 +367,257 @@ cp /etc/kubernetes/admin.conf $HOME/
...
+  vault_script_autounseal: true # enable automatic unseal vault at the start of the service, shouldn't be used at production
+  vault_script_autoconfiguration: true # enable automatic configuration of Hashicorp Vault
```
Comment (mkyc) [inline docs/home/howto/SECURITY.md:396]: I believe that should be explained more. What auto configuration is doing? What is being configured?

File: docs/home/howto/SECURITY.md [hunk 3]
```diff
@@ -367,3 +367,257 @@ cp /etc/kubernetes/admin.conf $HOME/
...
+
+Epiphany besides two already included in vault policies (root and default) provides two additional predefined policies:
+
+- admin - policy granting administration privileges, have sudo permission on Vault system endpoints
+- provisioner - policy granting permissions to create user secrets, adding secrets, enable authentication methods, but
+  without access to Vault system endpoints
+
+### Manual unsealing of the Vault
+
+When Hashicorp Vault starts it starts in sealed mode. This mean that Vault data is encrypted and Vault needs to generate key
```
Comment (mkyc) [inline docs/home/howto/SECURITY.md:443]: I would rephrase it a bit. Something like: "By design Hashicorp Vault starts in sealed mode."

Comment (mkyc) [inline docs/home/howto/SECURITY.md:443]: `data is encrypted and Vault needs to generate key` ... isn't it bit incorrect? I think it should be: `data is encrypted and operator needs to provide unsealing key` or something similar.

File: docs/home/howto/SECURITY.md [hunk 4]
```diff
@@ -367,3 +367,257 @@ cp /etc/kubernetes/admin.conf $HOME/
...
+to decrypt and access data.
+
+Vault can be unsealed manually using command:
+
+```bash
+vault operator unseal
+```
+
+and passing three unseal keys from /opt/vault/init.txt file. In the future number of keys will be defined from the level
```
Comment (mkyc) [inline docs/home/howto/SECURITY.md:452]: `In the future releases, number ...`

File: docs/home/howto/SECURITY.md (hunk 1 above)
Comment (erzetpe) [reply docs/home/howto/SECURITY.md:394]: Additional surface of attack, so when disabled more secure.
